### ISSUE TITLE
add ability to modify committer data

### ISSUE THREAD

**Issue by desrever-nu** [rtyley/bfg-repo-cleaner#91]
Another painful operation that is overly complex with filter is rewriting git history to edit/replace committers .

**Comment by rtyley** [comment]
Yeah, I wonder if the simplest interface would be to just bake-in the malmap file.

I haven't tried it, but I'm guessing that filter-branch may make replacing committer details fiddly, but it's probably reasonably fast, as it doesn't involve operating on the file-tree.

I've got a lot of things to work on at the moment :-)
